cypress/e2e/michael.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('drag', () => {
+    xit('click twice', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72954');
+        cy.get('.btn').click().click()
+    });
+
+    xit('41041', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41041')
+        cy.get('input[name="resulttext"]')
+            .type('{{}Click}')
```

Comment:
type('{Click}', {parseSpecialCharSequences: false})

is a better approach here for readability.